src/args.ts

```diff
@@ -0,0 +1,18 @@
+import { Command, InvalidArgumentError } from 'commander';
+
+const validLang = (value) => {
+	const validLangs = ['en', 'es', 'jp', 'zh', 'de', 'fr', 'it'];
```

Comment:
```suggestion
	const validLangs = ['en', 'es', 'jp', 'zh', 'de', 'fr', 'it', 'pt-br'];
```
It would be good if you add Brazilian Portuguese as an available language, as it is spoken by more than 200 million people.

Reply:
I think you should also set "en" as a default value

Reply:
@RigottiG thanks for your comments! I added just `pt` because I'm using ISO 639-1 for the codes.